packages/eslint-plugin-udemy/rules/no-then/index.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+module.exports.rules = {
+    'no-then': {
+        create(context) {
+            return {
+                MemberExpression(node) {
+                    if (node.property && node.property.name === 'then') {
+                        context.report({
+                            node,
+                            message: 'Found `then` used in a spec. Please use async / await instead.',
```

Comment:
The error message talks about specs, but the rule is not specific to specs.

Reply:
Perhaps you should change the error to:

...You should usually use async / await instead.